pkgs/top-level/all-packages.nix

```
@@ -791,6 +791,9 @@ in
   bitwarden_rs = callPackage ../tools/security/bitwarden_rs {
     inherit (darwin.apple_sdk.frameworks) Security CoreServices;
   };
+  bitwarden_rs-sqlite = bitwarden_rs.override { dbBackend = "sqlite"; };
```

This makes it a bit easier to read:

```suggestion
  bitwarden_rs-sqlite = bitwarden_rs;
```

---

I thought it best to just be explicit about it, but its true that most packages denote the default like that there as well (besides the package definition itself `dbBackend ? "sqlite"`), so I will change it.